src/components/JustNotSorry.js

```diff
@@ -128,12 +129,14 @@ class JustNotSorry extends Component {
 
       this.setState((state) => {
         let warnings = state.warnings;
```

Comment:
`let warnings = state.warnings;` isn't needed anymore.  You can add a `const` to the next line instead.